docs/Manual/Programs/Starter/Architecture.md

```diff
@@ -113,53 +113,53 @@ arangodb --starter.mode=cluster --starter.join=hostA:8528,hostB:8528,hostC:8528
 # this command is run on hostA, hostB and hostC.
 ```
 
-The state of the cluster (of Starters) is stored in a configuration file called
-`setup.json` in the data directory of every Starter and the ArangoDB
-agency is used to elect a master among all Starters.
+The state of the _Cluster_ (of _Starters_) is stored in a configuration file called
```

Comment:
I don't like the `_Cluster_` change, because it put more emphasis on it and is even more likely to be confused with an ArangoDB cluster. This is a cluster of starters.